templates/back-office/transfer-packing-slip.hypr

```diff
@@ -1,5 +1,86 @@
 ﻿{% extends "back-office/base" %}
 {% block body-content %}
+
+{% if model.packages %} 
+
+{% for packages in model.packages %}
```

Comment:
Please rename `packages` to its singular form, `package`

e.g.` {% for package in model.packages %}`